open-craft/sprints
# [BB-2353] Update dependencies

This is just a clean bump of all minors.

## Testing instructions 
1. Set `.env` file (you can find it in a comment on the ticket).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create superuser with `docker-compose -f local.yml exec django python manage.py createsuperuser`
1. Run migrations with `docker-compose -f local.yml migrate`.
1. Navigate to `frontend` directory and run `npm i && npm start`.
1. Log in.
1. Confirm that the frontend doesn't explode.
